playbooks/install-oauth-on-master.yaml

```diff
@@ -0,0 +1,38 @@
+---
+- name: Install RequestHeader Oauth Config in master-config.yaml
+  hosts: masters
+  tasks:
+    - name: remove existing HTPasswd provider
```

Comment:
shouldn't this be an option? I am thinking the case where you are still in troubleshoot mode and using HTPasswd as the way in.

Reply:
I tried this and found HTPasswd was not an option when using RequestHeader provider (on 3.9).  I'll try again on 3.11 and see if it works there.  Have you gotten it to work?  I know HTPasswd works fine when coupled with the Oauth provider with RH-SSO.

Reply:
Interestingly enough it worked now on 3.11.  Maybe there was a limitation before at some point, so I'll add some code for conditionals and document it.  There is also a revert playbook in the need one has to go back to only HTPasswd